src/Microsoft.ML.PipelineInference/PipelinePattern.cs

```diff
@@ -21,16 +21,24 @@ public sealed class PipelineResultRow
         {
             public string GraphJson { get; }
             public double MetricValue { get; }
+            public double TrainingMetricValue { get; }
```

Comment:
>TrainingMetricValue [](start = 26, length = 19)

can you add some comments to this property. What's the difference between MetricValue and TrainingMetricValue?

Reply:
I've added some comments above the properties, explaining them.

Reply:
XML comments are slightly better.

---
In reply to: [188105054](https://github.com/dotnet/machinelearning/pull/152#discussion_r188105054) [](ancestors = 188105054)

Reply:
I'll make that change.